src/components/Overlays/Overlays.fixture.tsx

```
@@ -0,0 +1,103 @@
+  const buttonRef = useRef<HTMLButtonElement>(null);
+
+  return (
+    <div>
+      <h3>Tooltip Examples</h3>
+
+      <div style={{ marginBottom: 20 }}>
+        <h4>Children Trigger</h4>
+        <p>
+          some text ...
+          <Tooltip content={'This is a tooltip with children trigger.\n ' + FILLER_TEXT}>
+            <button>Hover me (children)</button>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)